examples/hot_reload_tide/Cargo.toml

```diff
@@ -11,4 +11,4 @@ tide = "0.16.0"
 async-std = { version = "1.6.0", features = ["attributes"] }
 serde_json = "1.0"
 serde = "1.0.115"
-notify = { path = "../..", features = ["serde"] }
+notify = { version = "5.0.0-pre.10", features = ["serde"] }
```

Comment:
Can you add this also to the release_checklist.md ? Because we've to change/check this one too.

Reply:
Sure, added.